modules/odo-command-changes.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-migration-guide.adoc
+
+:_content-type: REFERENCE
+[id="odo-command-changes_{context}"]
+
+= Commands added, modified or removed in v3
+
+The following table contains a list of `odo` commands that have been added, modified or removed in v3. An *X* indicates that the command is not present in that version.
```

Comment:
Missing comma: `...command that have been added, modified, or removed in v3.`

Reply:
Yes (I wish Vale would catch this)